REQUIRE

```diff
@@ -1,3 +1,3 @@
 julia 0.5
-OpenCL
+OpenCL 0.5.2
```

Comment:
Minimum versions aren't for "this is what I tested" either. CI results record that. Minimum versions are for recording the lowest version that could possibly work.

But yes, CLBLAS master does pass tests against OpenCL 0.5.0 on my system.